src/sentry/incidents/models/alert_rule_activations.py

```diff
@@ -63,7 +65,9 @@ class AlertRuleActivations(Model):
     # metric value represents the query results at the end of the activated time window.
     # since activated alerts are not run on a continuously shifting time window, the value
     # of this metric value will only continually tick upwards until the monitor window has expired.
-    metric_value = models.FloatField()
+    metric_value = models.FloatField(null=True)
+    # query_subscriptions are cleaned up after every run, so we keep reference to the ids rather than foreign keying
+    query_subscription_id = models.IntegerField()
```

Comment:
I think you need a separate migration to add this

Reply:
yes ma'am you are correct